ClosedXML/XLHelper.cs

```diff
@@ -28,6 +28,11 @@ public static class XLHelper
         internal static readonly NumberStyles NumberStyle = NumberStyles.AllowDecimalPoint | NumberStyles.AllowLeadingSign | NumberStyles.AllowLeadingWhite | NumberStyles.AllowTrailingWhite | NumberStyles.AllowExponent;
         internal static readonly CultureInfo ParseCulture = CultureInfo.InvariantCulture;
 
+        internal static readonly Regex RCSimpleRegex = new Regex(
+            @"(r\d*c\d*)"
```

Comment:
I understand that you prepared this regex to check table names against it, and for that purpose it is sufficient. But since you use it in a public method `IsValidRCAddress` I would expect that it would treat relative addresses (`R[-1]`, `C[4]`, `R[1]C[-5]`...) as valid addresses too.

On the other hand, if #1201 will be accepted we can utilize parsing logic defined there.

Reply:
![image](https://user-images.githubusercontent.com/19576939/61042894-216b5a80-a3e6-11e9-92f3-0e4b807a5f9a.png)
Huh. It does not allow names _starting_ from something that can be a valid name.

I have to reconsider the PR.

Reply:
And for A1 it checks the entire name.

Reply:
I don't think we have to replicate Excel exactly. Even LibreOffice has different rules (less strict).